usr/src/uts/armv8/io/simple-bus.c

```diff
@@ -523,15 +523,8 @@ smpl_intr_ops(dev_info_t *pdip, dev_info_t *rdip, ddi_intr_op_t intr_op,
 				return (DDI_FAILURE);
 			}
 			kmem_free(irupts_prop, irupts_len);
-			switch (grp) {
-			case 1:
-				hdlp->ih_vector = vec + 16;
-				break;
-			case 0:
-			default:
```

Comment:
Interesting that the default case used to be SPI and now it is PPI, in contrast with cbe.c which was the other way round.

Reply:
Yes.  I'm pretty sure that what I did matched what I think we're trying to achieve here, but I should re-check this.